src/rdymesh.c

```diff
@@ -1071,6 +1071,78 @@ static PetscErrorCode CreateCellConnectionVector(DM dm, RDyMesh *mesh) {
   PetscFunctionReturn(PETSC_SUCCESS);
 }
 
+/// Determines the max number of edges/vertices per cell and max number of
+/// cells/edges per vertices
+/// @param [in] dm A PETSc DM object
+/// @param [inout] mesh A pointer to an RDyMesh that is updated
+/// @return PETSC_SUCCESS on success
+PetscErrorCode DetermineMaxAttributesForCellsAndVertices(DM dm, RDyMesh *mesh) {
```

Comment:
This can also be marked `static` if it's not used anywhere else.

Reply:
Good suggestion. I will update the PR.